source/release/4.5.0/index.md

```diff
@@ -72,69 +76,72 @@ dnf config-manager --set-enabled ovirt-45-upstream-testing
 #### oVirt Engine
 
  - [BZ 2056126](https://bugzilla.redhat.com/show_bug.cgi?id=2056126) **[RFE] Extend time to warn of upcoming certificate expiration**
-   RHV Manager 4.4 SP1 certificate expiration check will warn of upcoming certificate expiration earlier:
+   oVirt Engine 4.5 certificate expiration check will warn of upcoming certificate expiration earlier:
 
 1. If a certificate is going to expire in the upcoming 120 days, then WARNING event is raised in the audit log
 2. If a certificate is going to expired in the upcoming 30 days, then ALERT event is raised in the audit log
 
-This is check is covering internal RHV certificates (for example certificate for RHVM &lt;-&gt; hypervisor communication), but it doesn't check for custom certificates configured for HTTPS access to RHVM as configured according to https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html/administration_guide/appe-red_hat_enterprise_virtualization_and_ssl#Replacing_the_Manager_CA_Certificate
+This is check is covering internal oVirt certificates (for example certificate for oVirt Engine &lt;-&gt; hypervisor communication), but it doesn't check for custom certificates configured for HTTPS access to oVirt Egnine as configured according to https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html/administration_guide/appe-red_hat_enterprise_virtualization_and_ssl#Replacing_the_Manager_CA_Certificate
```

Comment:
> This is check is covering
I think you might mean "This checks for internal oVirt certificates ..."

>oVirt Egnine
Typo: Engine

Reply:
Issue here is that this note is autogenerated from https://bugzilla.redhat.com/show_bug.cgi?id=2056126 doctext and manually rebranded. I need to find a way to automatically update this.

Reply:
working on this here: https://github.com/oVirt/releng-tools/pull/43